src/Jackett.Common/Indexers/EliteTracker.cs

```diff
@@ -206,6 +208,13 @@ public EliteTracker(IIndexerConfigurationService configService, WebClient webCli
                     release.Peers = ParseUtil.CoerceInt(Leechers.TextContent) + release.Seeders;
                     release.Grabs = ParseUtil.CoerceLong(Grabs.TextContent);
 
+                    if (TorrentHTTPSMode)
+                    {
+                        var linkHttps = Row.QuerySelector("td:nth-child(4)").QuerySelector("a").GetAttribute("href");
+                        var idTorrent = ParseUtil.GetArgumentFromQueryString(linkHttps, "id");
+                        release.Link = new Uri($"https://elite-tracker.net/download.php?id={idTorrent}&type=ssl");
```

Comment:
Please use the SiteLink variable instead of hard coding https://elite-tracker.net/